arm-sql/compositeSql.json

```diff
@@ -13,8 +13,6 @@
     "./2014-04-01/swagger/servers.json",
     "./2014-04-01/swagger/sql.core.json",
     "./2014-04-01/swagger/databaseSecurityAlertPolicies.json",
-    "./2015-05-01-preview/swagger/blobAuditingPolicies.json",
```

Comment:
Had offline chat with reviewers and we think this should be okay. I need to point out however, that these would be breaking changes.